src/components/welcome/MyAccounts/useTrackedSafesCount.ts

```diff
@@ -34,6 +59,17 @@ const useTrackSafesCount = (
     }
   }, [isLoginPage, ownedSafes, wallet])
 
+  useEffect(() => {
+    const totalSafesPinned = pinnedSafes?.reduce(
+      (prev, current) => prev + (isMultiChainSafeItem(current) ? current.safes.length : 1),
+      0,
+    )
+    if (wallet && !isPinnedSafesTracked && pinnedSafes && pinnedSafes.length > 0 && isLoginPage) {
+      trackEvent({ ...OVERVIEW_EVENTS.TOTAL_SAFES_PINNED, label: totalSafesPinned })
```

Comment:
For visibility: I noticed there is now an overlap of watchlist safes tracking and pinned safes tracking i.e. the watchlist safes contain the pinned safes + any safe that has a name while the pinned safes only contain pinned safes.

I would suggest filtering pinned safes from watchlisted safes to account for this e.g. if I have 3 pinned safes and 2 additional watched safes we would track 3 for pinned and 2 for watchlisted instead of 5 for watchlisted.

Reply:
makes sense, thanks!